_tutorials/import-export-data.html

```diff
@@ -32,12 +32,15 @@ <h3 id="import-youtube">Import YouTube subscriptions</h3>
         <figure class="vertical">
             <img src="/img/screenshots/shot_subscriptions_export_light.png" class="img-responsive screenshot" />
             <figcaption>
-                <ul class="inside">
-                    <li>Go to the <a href="https://www.youtube.com/subscription_manager?action_takeout=1">YouTube subscriptions manager</a>.</li>
+                <ol>
+                    <li>Go to the <a href="https://takeout.google.com/takeout/custom/youtube">Google takeout manager</a>.</li>
                     <li>Log in if asked.</li>
-                    <li>A download will start. It contains your exported YouTube subscriptions.</li>
-                    <li>Go back to NewPipe and tap on the import file at the bottom of the screen. Now, select your downloaded YouTube export file.</li>
-                </ul>
+                    <li>Click on "All data included", then on "Deselect all", then select only "subscriptions" and click "OK".</li>
+                    <li>Click on "Next step" and then on "Create export".</li>
+                    <li>Click on the "Download" button after it appears.</li>
+                    <li>From the downloaded takeout <code>zip</code> extract the <code>.json</code> file. It is usually located under <pre style="white-space: normal">YouTube and YouTube Music/subscriptions/subscriptions.json</pre>.</li>
+                    <li>Go back to NewPipe, tap on "Import from > YouTube", hit "import file" at the bottom of the screen and select the .json file.</li>
```

Comment:
Oh, sorry, I seem to have mis-sent the previous review. I meant that this should be changed and then everything is good.
```suggestion
                    <li>Go back to NewPipe, tap on "Import from > YouTube", hit "import file" at the bottom of the screen and select the <code>.json</code> file.</li>
```

Reply:
Done in 84d7cdb2d12e9a6de834eae6539dcef31533fb75